packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -59,9 +45,9 @@ drag-and-drop functionality (signalized by the `has-drag-and-drop` state class
 on the root element).
 
 ```html
-<div class="FileUploaderInput">
+<div class="FileUploaderInput" data-element="wrapper">
   <label for="fileUpload" class="FileUploaderInput__label">Label</label>
-  <div class="FileUploaderInput__dropZone">
+  <div class="FileUploaderInput__dropZone" data-element="dropZone">
```

Comment:
One more thing: why are you using `data-element` attributes? Can't you just use the classes? It seems to me like a duplication of selectors. 🤔

Reply:
@adamkudrna or @crishpeen will give you the answer

Reply:
no class selectors, please. They might be prefixed and they are subject to change (without releasing it might modify the JS)